docs/user/custom-resources/apigateway/apigateway-v1beta2-adr.md

```diff
@@ -0,0 +1,118 @@
+| **rules.service**               |  **NO**   | Services definitions at this level have higher precedence than the Service definition at the **spec.service** level.                                                                                                                                                                                                                         |
+| **rules.path**                  |  **YES**  | Specifies the path of the exposed Service.                                                                                                                                                                                                                                                                                                   |
+| **rules.methods**               |  **NO**   | Specifies the list of HTTP request methods available for **spec.rules.path**. The list of supported methods is defined in [RFC 9910: HTTP Semantics](https://www.rfc-editor.org/rfc/rfc9110.html) and [RFC 5789: PATCH Method for HTTP](https://www.rfc-editor.org/rfc/rfc5789.html).                                                        |
+| **rules.mutators**              |  **NO**   | Specifies the list of the Istio mutators.                                                                                                                                                                                                                                                                                                    |
+| **rules.accessStrategies**      |  **YES**  | Specifies the list of access strategies. Supported are `no_auth`, `extAuth` and `jwt` as [Istio](https://istio.io/latest/docs/tasks/security/authorization/authz-jwt/) access strategy.                                                                                                                                                      |
+| **rules.timeout**               |  **NO**   | Specifies the timeout, in seconds, for HTTP requests made to **spec.rules.path**. The maximum timeout is limited to 3900 seconds (65 minutes). Timeout definitions set at this level take precedence over any timeout defined at the **spec.timeout** level.                                                                                 |
+
+### Examples
+
+- Multiple hosts with external authorizer:
+```yaml
+apiVersion: gateway.kyma-project.io/v1beta2
+kind: APIRule
+metadata:
+  name: service-config
+spec:
+  gateway: kyma-system/kyma-gateway
+  hosts:
+    - api1.example.com
+    - api2.example.com
+  service:
+    name: httpbin
+    port: 8000
+  rules:
+    - path: /headers
+      methods: ["GET"]
+      mutators:
```

Comment:
Is there a reason why we add the mutators in the examples? For me it looks like they are not relevant for the examples and we should rather have more concrete examples with mutators if we want to add them.